gradle.properties

```diff
@@ -0,0 +1,28 @@
+# IntelliJ Platform Artifacts Repositories -> https://plugins.jetbrains.com/docs/intellij/intellij-artifacts.html
+pluginGroup=com.redhat.devtools.intellij
+pluginName=Language Server Protocol Client for IntelliJ
+pluginRepositoryUrl=https://github.com/redhat-developer/lsp4ij
+# SemVer format -> https://semver.org
+# NO SPACES AROUND THE EQUALS SIGN!!
+pluginVersion=0.0.1
+# Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
+pluginSinceBuild=222
+#pluginUntilBuild=233.*
+# IntelliJ Platform Properties -> https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#configuration-intellij-extension
+platformType=IC
+platformVersion=2022.2.3
+# Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
+# Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
+platformPlugins=com.intellij.java, maven, gradle-java, properties, yaml, com.redhat.devtools.intellij.telemetry:1.0.0.44
```

Comment:
com.intellij.java, maven, gradle-java, properties, yaml should be removed

Reply:
removed